ui/src/app/features/app-user-list/application-user-list.component.ts

```diff
@@ -125,20 +129,22 @@ export class ApplicationUserListComponent implements OnInit, OnDestroy {
 
     const dialogSubscription = dialog.afterClosed().subscribe(({ userEmail, role }) => {
       if (userEmail && role) {
-        this.appUserListFacade.inviteUser(userEmail, role).subscribe(() => this.openEmailNotification(userEmail));
+        this.appUserListFacade.inviteUser(userEmail, role).subscribe(
```

Comment:
need to dispatch only one action - 'invite user' here. Notifications actions should be handled in effects